### ISSUE TITLE
2 Alignment steps

### ISSUE THREAD

**Issue by shashwatsahay** [fulcrumgenomics/fgbio#970]
Hi 

Is it necessary to align the sequence twice? My files are quite large takes approx 8 hours for one run of bwa mem. 

Can we skip the alignement step in Step 1.2 of https://github.com/fulcrumgenomics/fgbio/blob/main/docs/best-practice-consensus-pipeline.md and directly proceed to GroupReadsByUmi  and start the downstreams Phase 2 steps.

Any insights into why the prealignment is necessary will be helpful

**Comment by nh13** [comment]
Yes it is required, do you have someone that can help you at your organization to help mentor you why?   We're available for consulting too: https://www.fulcrumgenomics.com/
